<ExceptionInfo ZeroDivisionError('division by zero') tblen=1><class 'ZeroDivisionError'>[<TracebackEntry F:\Projects\Python\pytest_t\tests\test_main_five_solo.py:22>]division by zero